docs/modules/bindings-specification/pages/message-passing-api.adoc

```diff
@@ -189,7 +192,17 @@ class Checksums {
   /// The sha-256 checksum of this dependency's metadata.
   sha256: String
 }
+
+class Http {
+  /// Certificates to trust when making HTTP requests.
+  ///
+  /// If [null], Pkl will trust its own built-in certificates.
+  caCertificates: Binary?
```

Comment:
At first, I considered accepting a path to a cert file here. However, this doesn't work so well if a client wants to trust system certs on macOS, because macOS's system certs are managed by Security.framework, and not stored as files on disk. Accepting certs directly as binary seems like a better choice for the message passing API.